### PR TITLE
Some minor enhancements

### PR DESCRIPTION
**Pull request by paulstapor**
1. I added an option for profile integration (how much remaining gradient should be tolerated), to make it more customizable
2. I made meigoDummy a bit more flexible, since I hacked Meigo a little bit to make it use hessians in the case...
3. I added an additional check for inf also in gradients in the wrapper to give back inf in the pbjective, if this occurs.

### CODE REVIEW

File: private/objectiveWrap.m [hunk 1]
```diff
@@ -58,6 +59,10 @@
                     case 'log-posterior'          , varargout = {-J,-G(I)};
                     case 'negative log-posterior' , varargout = { J, G(I)};
                 end
+                if any(any(isnan(G))) || any(any(isinf(G)))
```
Comment (dweindl) [inline private/objectiveWrap.m]: Would use [isfinite](https://de.mathworks.com/help/matlab/ref/isfinite.html) instead; this checks for `nan` and `inf` simultaneously. Same for Hessian and gradient below.

File: @PestoOptions/PestoOptions.m [hunk 2]
```diff
@@ -253,6 +253,7 @@
         % *     .MaxStep ... minimum step size of the solver
```
Comment (dweindl) [inline @PestoOptions/PestoOptions.m]: Should be .MinStep

File: private/objectiveWrap.m (hunk 1 above)
Comment (paulstapor) [inline private/objectiveWrap.m]: done

File: @PestoOptions/PestoOptions.m (hunk 2 above)
Comment (paulstapor) [inline @PestoOptions/PestoOptions.m]: done

**Pull request merged**